x-pack/plugins/observability/public/components/app/resources/index.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { EuiFlexGroup, EuiFlexItem, EuiListGroup, EuiTitle } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+
+const resources = [
+  {
+    iconType: 'documents',
+    label: i18n.translate('xpack.observability.resources.documentation', {
+      defaultMessage: 'Documentation',
+    }),
+    // TODO: caue what's the url?
+    href: 'https://www.elastic.co',
```

Comment:
@bmorelli25 Can you provide the correct URL when the docs page is ready?

Reply:
The documentation URL is `https://www.elastic.co/guide/en/observability/current/observability-ui.html`. Which, _I think_, would be this:

```
<ElasticDocsLink
  section="/observability"
  path="/observability-ui.html"
>
```

But I'm not sure how that fits in here 😬